Sources/MapboxCoreNavigation/CoreNavigationNavigator.swift

```diff
@@ -36,14 +36,14 @@ class Navigator {
         .init(setRoutesHandler: { [weak self] route, legIndex, completion in
             self?.navigator.setPrimaryRouteForRoute(route, legIndex: legIndex) { [weak self] result in
                 if result.isValue() {
-                    let routeInfo = result.value as! RouteInfo
```

Comment:
NN now exposes actual generics instead of AnyObject, so no need to cast here and in other places